tutoraspects/templates/aspects/build/aspects-superset/openedx-assets/assets/dashboards/Course_Dashboard.yaml

```diff
@@ -1366,6 +1357,6 @@ position:
     - TAB-_Ey4nPhFr
     type: TABS
 published: true
-slug: course-dashboard-v1
+slug: course-dashboard
```

Comment:
This change breaks embedding, can we leave it as it was so we don't have to change the uuids everywhere?

Reply:
It shouldn't break it, we need to update the settings instead to match the new value

Reply:
It causes them to 404, so either we need to change it back or change it in the settings here and in platfform-plugin-aspects